events/syscall/potential_local_privillege_escalation_via_env_var_misuse.go

```diff
@@ -18,26 +18,24 @@ limitations under the License.
 package syscall
 
 import (
-    "os"
-    "os/exec"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
 
-    "github.com/falcosecurity/event-generator/events"
+	"github.com/falcosecurity/event-generator/events"
 )
 
-var _ = events.Register(PotentialLocalPrivillegeEscalation)
+var _ = events.Register(PotentialLocalPrivilegeEscalationViaEnvironmentVariablesMisuse)
```

Comment:
The file name does not match. 
Note the typo :point_down: 
potential_local_privil**l**ege_escalation_via_env_var_misuse.go

Reply:
renamed file to `potential_local_privilege_escalation_via_environment_variables_misuse.go`